core/pom.xml

```diff
@@ -772,7 +772,6 @@ THE SOFTWARE.
           <forkCount>0.5C</forkCount>
           <reuseForks>true</reuseForks>
           <argLine>-noverify</argLine> <!-- some versions of JDK7/8 causes VerifyError during mock tests: http://code.google.com/p/powermock/issues/detail?id=504 -->
-          <trimStackTrace>false</trimStackTrace> <!-- SUREFIRE-1226 workaround -->
```

Comment:
Related: without moving this configuration up, critical information was being suppressed from test failures in the `cli` module.